resources/bastion/quotes-app.yaml

```
@@ -0,0 +1,72 @@
+---
+apiVersion: v1
+kind: List
+metadata: {}
+items:
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      labels:
+        app: famous-quotes
+      name: famous-quotes
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: famous-quotes
+      template:
+        metadata:
+          labels:
+            app: famous-quotes
+        spec:
+          containers:
+            - name: famous-quotes
+              image: quay.io/redhattraining/famous-quotes:2.1
+              ports:
+                - containerPort: 8000
+                  protocol: TCP
+              env:
+                - name: QUOTES_HOSTNAME
+                  value: (( add-db-endpoint-here }}
```

In all the OpenShift courses, we use the `<CHANGE_ME>` tag to instruct students to change something.
So if you plan to ask students to edit that file and change that value, the use the placeholder:

```suggestion
                  value: <CHANGE_ME>
```